CommonTools/RecoAlgos/plugins/PtMinMuonCountFilter.cc

```diff
@@ -16,5 +16,14 @@ typedef ObjectCountFilter<
           reco::MuonCollection, 
           PtMinSelector
         >::type PtMinMuonCountFilter;
+template <>
+struct SelectorFillDescriptions<PtMinMuonCountFilter> {
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
```

Comment:
responding to https://github.com/cms-sw/cmssw/pull/21241#issuecomment-346572064 here:
is this layer actually needed?

I was thinking of something like this
https://github.com/cms-sw/cmssw/compare/master...slava77:CMSSW_10_0_0_pre1/evSelector-fillDescSpec-example
[I've reused your DoFillAsUnknown]

Reply:
Apparently no, the layer is not needed. Somehow I had omitted the fact that single member functions can be specialized, making your suggestion even simpler implementation.

(the pros+cons vs. #21430 in the PR description still holds)

Reply:
If I'm not mistaken, this PR (or my simpler solution) and #21430 can "coexist" at the end of specific module template instantiation.
The #21430 provides a default solution that may or may not be the best and the final instance can have a hand-made implementation of the fillDescriptions.
If we have enough time with #21241, I'd propose to close this,  make #21430 runnable and aim to merge that as a default solution.

Reply:
Indeed #21430 and this PR/Slava's solution can coexist. If we then take the route of "coexistence", does #21430 need the `fillDescriptions` customization mechanism (that allows overriding individual parameters), or is it sufficient to say that if user wants to customize anything (including the module label) (s)he must specify the full `fillDescriptions`?

I'm asking because removing the customization mechanism from #21430 would make it simpler (e.g. no need for SFINAE checks).

Reply:
it's hard to make comments on another PR. Can we move the discussion there?

Reply:
Sure.